app/src/main/java/com/android/sample/model/recipe/RecipeBuilder.kt

```diff
@@ -145,7 +145,7 @@ class RecipeBuilder {
   fun build(): Recipe {
     // Validation for essential fields
     require(name.isNotBlank()) { ERROR_STR_MEAL_BLANK }
-    require(instructions.isNotBlank()) { ERROR_STR_INSTR_BLANK }
+    require(instructions.isNotEmpty()) { ERROR_STR_INSTR_BLANK }
```

Comment:
Thanks, we will discuss it tomorrow, but I think that it comes from the `Instruction` class where in the `init` method, I check if the `description` parameter is not blank. I also think it will take 2 minutes to fix.